data/human/free worlds 1 start.txt

```diff
@@ -26,7 +26,7 @@ mission "Liberate Kornephoros"
 		event "prepare for battle of Kornephoros"
 		conversation
 			`The mood in the spaceport is solemn. No one has had time yet to count up the losses, on both sides of the battle, but it is likely that over a thousand lives were lost. And because the Navy is held in such high regard even here where Parliament and the Paradise Planets are viewed with disgust, many of the militia captains are almost as saddened by the loss of Navy lives as by the loss of some of their own.`
-			`	You find Tomek walking among the fleet, conversing with a large group of captains and crew members and assessing the damage they have suffered. When he sees you he says, "Captain <last>! How is your <ship> holding together?"`
+			`	You find Tomek walking among the fleet, conversing with a large group of captains and crew members and assessing the damage they have suffered. When he sees you he says, "Captain <last>! How is the <ship> holding together?"`
```

Comment:
"The" doesn't make much sense to me because a ship name could start with the, so you could have a line like "How is the The Doomship holding together?" which really isn't how someone would ask that. Leaving "your" there for "your The Doomship" seems odd too, and I think "your" would made more sense if it were preceding the ship model, but this is the ship name. So really, I think we should just leave this as "How is `<ship>`..."
```suggestion
			`	You find Tomek walking among the fleet, conversing with a large group of captains and crew members and assessing the damage they have suffered. When he sees you he says, "Captain <last>! How is <ship> holding together?"`
```

Reply:
Alright